dropwizard-auth/src/main/java/io/dropwizard/auth/AuthDynamicFeature.java

```diff
@@ -0,0 +1,26 @@
+package io.dropwizard.auth;
+
+import com.google.common.base.Function;
+import org.glassfish.jersey.server.model.AnnotatedMethod;
+import javax.ws.rs.container.DynamicFeature;
+import javax.ws.rs.container.ResourceInfo;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class AuthDynamicFeature implements DynamicFeature {
+    final private AuthHandler authHandler;
+
+    public AuthDynamicFeature(AuthHandler authHandler) {
+        this.authHandler = authHandler;
+    }
+
+    @Override
+    public void configure(ResourceInfo resourceInfo, FeatureContext context) {
+        AnnotatedMethod am = new AnnotatedMethod(resourceInfo.getResourceMethod());
+
+        if (am.isAnnotationPresent(Auth.class)) {
```

Comment:
The auth stuff in my project is already going to need to be refactored to move to DW 0.8.0. Having this change at the same time as the other 0.8.x stuff is better than making devs make sweeping revisions everything twice in short succession.

+1